source/appModules/soffice.py

```diff
@@ -101,6 +105,37 @@ def _getFormatFieldAndOffsets(self,offset,formatConfig,calculateOffsets=True):
 		if backgroundColor:
 			formatField['background-color']=colors.RGB.fromString(backgroundColor)
 
+		if offset == 0:
+			# Only include the list item prefix on the first line of the paragraph.
+			numbering = formatField.get("Numbering")
+			if numbering:
+				formatField["line-prefix"] = numbering.get("NumberingPrefix") or numbering.get("BulletChar")
+
+		return formatField
+
+	def _getFormatFieldAndOffsets(self, offset, formatConfig, calculateOffsets=True):  # noqa: C901

```

Comment:
could this function get type annotations as well please

Reply:
Done in new version.